osm2pgsql.lua

```diff
@@ -499,128 +499,116 @@ function osm2pgsql.process_node(object)
-        output_hstore['place:TW'] = 'region'
-        output_hstore['place:UA'] = 'region'
-        output_hstore['place:VN'] = 'region'
+        output_hstore['place'] = nil
     end
--- Turn off Nagorno-Karabakh label for most countries
+-- Turn off Nagorno-Karabakh label
     if object.tags.place and object.tags.wikidata == 'Q2397204' then
-        output_hstore['place:AR'] = 'region'
-        output_hstore['place:BD'] = 'region'
-        output_hstore['place:BR'] = 'region'
-        output_hstore['place:CN'] = 'region'
-        output_hstore['place:DE'] = 'region'
-        output_hstore['place:EG'] = 'region'
-        output_hstore['place:ES'] = 'region'
-        output_hstore['place:FR'] = 'region'
-        output_hstore['place:GB'] = 'region'
-        output_hstore['place:GR'] = 'region'
-        output_hstore['place:ID'] = 'region'
-        output_hstore['place:IL'] = 'region'
-        output_hstore['place:IN'] = 'region'
-        output_hstore['place:IT'] = 'region'
-        output_hstore['place:JP'] = 'region'
-        output_hstore['place:KO'] = 'region'
-        output_hstore['place:MA'] = 'region'
-        output_hstore['place:NL'] = 'region'
-        output_hstore['place:NP'] = 'region'
-        output_hstore['place:PK'] = 'region'
-        output_hstore['place:PL'] = 'region'
-        output_hstore['place:PS'] = 'region'
-        output_hstore['place:PT'] = 'region'
-        output_hstore['place:RU'] = 'region'
-        output_hstore['place:SA'] = 'region'
-        output_hstore['place:SE'] = 'region'
-        output_hstore['place:TR'] = 'region'
-        output_hstore['place:TW'] = 'region'
-        output_hstore['place:US'] = 'region'
-        output_hstore['place:VN'] = 'region'
+        output_hstore['place'] = nil
     end
--- Turn off Somaliland label for most countries
+-- Turn off Somaliland label
     if object.tags.place and object.tags.wikidata == 'Q34754' then
-        output_hstore['place:AR'] = 'region'
-        output_hstore['place:BD'] = 'region'
-        output_hstore['place:BR'] = 'region'
-        output_hstore['place:CN'] = 'region'
-        output_hstore['place:EG'] = 'region'
-        output_hstore['place:GR'] = 'region'
-        output_hstore['place:ID'] = 'region'
-        output_hstore['place:IL'] = 'region'
-        output_hstore['place:IN'] = 'region'
-        output_hstore['place:MA'] = 'region'
-        output_hstore['place:NP'] = 'region'
-        output_hstore['place:PK'] = 'region'
-        output_hstore['place:PL'] = 'region'
-        output_hstore['place:PS'] = 'region'
-        output_hstore['place:PT'] = 'region'
-        output_hstore['place:RU'] = 'region'
-        output_hstore['place:SA'] = 'region'
-        output_hstore['place:SO'] = 'region'
-        output_hstore['place:TR'] = 'region'
-        output_hstore['place:TW'] = 'region'
-        output_hstore['place:UA'] = 'region'
-        output_hstore['place:VN'] = 'region'
+        output_hstore['place'] = nil
```

Comment:
I think this should be place = country here, and then delete the row in NE "countries" table for Somaliland and it'll get exported as `unrecognized`.

Then all the other country overrides are deleted.